android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -124,7 +125,29 @@ private void showLocationsOnMap() {
             builder.include(marker.getPosition());
         }
 
-        handleSearchEvents();
+        LayoutInflater inflator = (LayoutInflater) getActivity()
+                .getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        View toolbarView = inflator.inflate(R.layout.map_toolbar, null);
+        ButterKnife.bind(this, toolbarView);
+
+        toolbar.setCustomView(toolbarView );
+
+        //Setting up AutoCompleteTextView with the locations
+        ArrayAdapter<String> adapter = new ArrayAdapter<>(getActivity(), android.R.layout.simple_dropdown_item_1line, searchItems);
+        textView.setAdapter(adapter);
+
+        textView.setOnItemClickListener((parent, view, position, id) -> {
+            String loc = adapter.getItem(position);
+            int pos = searchItems.indexOf(loc);
+            LatLng lng = new LatLng(mLocations.get(pos).getLatitude(), mLocations.get(pos).getLongitude());
+            mMap.animateCamera(CameraUpdateFactory.newLatLngZoom(lng, (float) Math.floor(mMap.getCameraPosition().zoom + 8)));
```

Comment:
Along with it please change icon of marker when item is selected and  showInfoWindow.

i think it will look good if there is no underline in auto complete text view and hint text color is changed